install-dependencies.sh

```diff
@@ -29,6 +29,7 @@ deb_deps=(
   clang
   curl
   git
+  gtk-doc-tools
```

Comment:
This actually installs `gtk-doc-tools` while the comment is saying that we introduce redirect to `/bin/true`